src-docs/src/views/text/text.js

```diff
@@ -18,89 +18,115 @@ export default () => (
     <OuiText grow={false}>
       <h1>This is Heading One</h1>
       <p>
-        Far out in the uncharted backwaters of the <a href="#">unfashionable</a>{' '}
-        end of the western spiral arm of the Galaxy lies a small unregarded
-        yellow sun. When suddenly some wild JavaScript code appeared!{' '}
+        Veniam et quis in dolor aliqua dolor laboris nostrud nostrud Lorem.
+        <a href="#">unfashionable</a> Irure et et adipisicing eu mollit. Ullamco
```

Comment:
unfashionable?

Reply:
This was ok'd by me as I gave the instruction to "keep links in tact"